lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -165,47 +165,4 @@ private static function remove_JSON_comments( $array ) {
 		return $array;
 	}
 
-	/**
```

Comment:
This is part of https://github.com/WordPress/gutenberg/pull/45969 To be removed when that PR lands.